server/ocsp.go

```diff
@@ -161,16 +163,43 @@ func (oc *OCSPMonitor) getRemoteStatus() ([]byte, *ocsp.Response, error) {
 	if config := opts.OCSPConfig; config != nil {
 		overrideURLs = config.OverrideURLs
 	}
-	getRequestBytes := func(u string, hc *http.Client) ([]byte, error) {
+	getRequestBytes := func(u string, reqDER []byte, hc *http.Client) ([]byte, error) {
```

Comment:
Do we need Get as well here? Do all OCSP support POST? Should we just support that?

Reply:
In theory they should, but the problem is "do they really, and do the applicaiton firewalls permit it", inside corporate environments.  So since we have been using GET, I think the goal is to make sure that worst case scenario, for things we don't control, we don't cause a regression in system behavior.

Reply:
They should support both, but we have used GETs only so far so in case there was a firewall or something depending on that behavior already we do the fallback just in case.